Example/Example/ViewController.swift

```diff
@@ -219,7 +219,23 @@ class RowsExampleViewController: FormViewController {
                         $0.value = 👦🏼
                         $0.selectorTitle = "Choose an Emoji!"
                     }
-            
+
+                <<< SectionedPushRow<String, Emoji>() {
```

Comment:
This is now removed, now it's just a normal PushRow and you can set closures to break options by sections in `onPresent` block